prisma/migrations/20230101120731_name_change/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `physicalAddressId` on the `Contact` table. All the data in the column will be lost.
+  - Added the required column `addressId` to the `Contact` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Contact" DROP CONSTRAINT "Contact_physicalAddressId_fkey";
+
+-- AlterTable
+ALTER TABLE "Contact" DROP COLUMN "physicalAddressId",
```

Comment:
question: is this just for updating our current data or is this meant to be run every time?

Reply:
Migrations are only run once on the database

Reply:
ohhhhh ok....that makes sense. I was thinking these were just interim migrations and then long-term migrations were idempotent. Ryokai 👍